Counter sensor won't compile with EVENTS_INTERRUPT_MODE=BOTH

Hi

I want to use a reed sensor, and am experimenting with a counter sensor to get instant notifications. The docs says:

> EVENTS_INTERRUPT_MODE | On which signal transition to react. 
> One of:  - RISING (default)  - FALLING  - BOTH

How ever when I compile with:

> build_flags = ${common.build_flags_1m0m} -DITEAD_S20 -DEVENTS_SUPPORT=1 -DEVENTS_PIN=1 -DEVENTS_PIN_MODE=INPUT_PULLUP -DEVENTS_INTERRUPT_MODE=BOTH

I get:

```Building firmware images...
...
/home/user/Downloads/git/espurna/code/espurna/sensor.ino: In function 'void _sensorLoad()':
<command-line>:0:23: error: 'BOTH' was not declared in this scope
/home/user/Downloads/git/espurna/code/espurna/sensor.ino:454:34: note: in expansion of macro 'EVENTS_INTERRUPT_MODE'
sensor->setInterruptMode(EVENTS_INTERRUPT_MODE);
^
*** [.pioenvs/itead-s20/src/espurna.ino.cpp.o] Error 1
================================= [ERROR] Took 3.76 seconds =================================
```

I tried looking at the code, but it's beyond me to figure out how or where BOTH should be defined.

Very grateful for the project! =)

--- a/code/espurna/config/sensors.h
+++ b/code/espurna/config/sensors.h
@@ -295,7 +295,7 @@
 #endif
 
 #ifndef EVENTS_INTERRUPT_MODE
-#define EVENTS_INTERRUPT_MODE           RISING  // RISING, FALLING, BOTH
+#define EVENTS_INTERRUPT_MODE           RISING  // RISING, FALLING, CHANGE
 #endif
 
 #define EVENTS_DEBOUNCE                 50      // Do not register events within less than 50 millis
@@ -318,7 +318,7 @@
 #endif
 
 #ifndef GEIGER_INTERRUPT_MODE
-#define GEIGER_INTERRUPT_MODE           RISING  // RISING, FALLING, BOTH
+#define GEIGER_INTERRUPT_MODE           RISING  // RISING, FALLING, CHANGE
 #endif
 
 #define GEIGER_DEBOUNCE                 25      // Do not register events within less than 25 millis.

